src/lang/Messages.properties

```diff
@@ -894,7 +895,6 @@ context.import.label.context	   = Context:
 context.import.label.dir		   = Directory:
 context.import.label.file		   = File:
 context.import.label.overwrite	   = Overwrite:
```

Comment:
Worth fixing these as well? (Note that in the export dialogue the other resource keys, labels and error messages, are also using import.)